### PR TITLE
TIP-5900: Change dashboard

### PR DESCRIPTION
**Pull request by pierallard**
screenshot from 2016-08-01 12 24 30

TODO: change the commits name to always have "PIM-5900".

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | n
| Changelog updated                 | n
| Review and 2 GTM                  | TODO
| Micro Demo to the PO (Story only) | n
| Migration script                  | n
| Tech Doc                          | n

### CODE REVIEW

**Comment by nidup** [comment]
Screenshot plz :)

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<% var chunkedChannels = _.toArray(_.groupBy(data, function(element, index) { return Math.floor(index / options.channelsPerRow); })); %>
```
Comment (grena) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html]: I'd like to see this in the JS part, but I understand it's a display concern

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                <%= _.__("pim_dashboard.widget.last_operations.date") %>
+            </th>
+            <th>
+                <%= _.__("pim_dashboard.widget.last_operations.type") %>
+            </th>
+            <th>
+                <%= _.__("pim_dashboard.widget.last_operations.profile name") %>
+            </th>
+            <th>
+                <%= _.__("pim_dashboard.widget.last_operations.status") %>
```
Comment (grena) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: simple quotes please :)

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+    <% _.each(data, function (operation) { %>
+        <tr>
+            <td> <%= operation.date %></td>
+            <td> <%= _.__("pim_dashboard.widget.last_operations.job_type."+operation.type) %></td>
```
Comment (grena) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: simple quotes

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <td>
+                <span class="label <%= operation.labelClass %>">
+                    <%= operation.statusLabel %>
+                </span>
+            </td>
+            <td class="text-right">
+                <a class="btn btn-large btn-download" href="javascript:void(0);"
+                data-id="<%= operation.id %>"
+                data-operation-type="<%= operation.type %>">
+                    <%= _.__("pim_dashboard.widget.last_operations.details") %>
```
Comment (grena) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: simple quotes

**Comment by grena** [comment]
Some comments to fix then GTM

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    }
+  }
+  .label, .btn {
+    font-size: @font-size-12;
+    .border-radius(6px);
+    text-shadow: none;
+    line-height: 34px;
+    height: 34px;
+    padding: 0 20px;
+    font-weight:normal;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: space after

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+  }
+
+  .channels {
+    display: flex;
+    justify-content: space-between;
+
+    .channel {
+      width: 300px;
+      display: inline-block;
+      *display: inline;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: why *?

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+      display:inline-block;
+      margin-left: 10px;
+      :first-child {
+        margin-right: 3px;
+      }
+    }
+    .last-version {
+      display: inline-block;
+      margin-left: 10px;
+      :first-child {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: &:first-child

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+  }
+  &.header {
+    background-color: @whiteDark;
+    padding: 15px;
+  }
+  .version-container {
+    .current-version {
+      display:inline-block;
+      margin-left: 10px;
+      :first-child {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: same

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+    }
+  }
+
+  background-color: white;
+  &.quick-launchpad > .container {
+    padding-top: 30px;
+  }
+  .quick-launchpad-widget {
+    margin-bottom: 20px;
+    &>div.clearfix {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: $ > div

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+    text-transform: uppercase;
+
+    img {
+      display:block;
+      margin: 0 auto;
+      padding-bottom: 10px;
+    }
+    .btn.shortcut {
+      display: inline-block;
+      *display: inline;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: same

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+    display: flex;
+    justify-content:space-between;
+    text-transform: uppercase;
+
+    img {
+      display:block;
+      margin: 0 auto;
+      padding-bottom: 10px;
+    }
+    .btn.shortcut {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: bank line before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 12]
```diff
@@ -0,0 +1,200 @@
+@import '../../pimui/css/less/variables';
+@import '../../pimui/css/less/bootstrap_mixins';
+
+.dashboard {
+  color: @akeneoSlateGrey;
+
+  .shortcuts {
+    display: flex;
+    justify-content:space-between;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: space before space-between

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+    text-transform: uppercase;
+
+    img {
+      display:block;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: space before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+    color: @color-new-purple-medium;
+    font-weight: normal;
+  }
+  &.header {
+    background-color: @whiteDark;
+    padding: 15px;
+  }
+  .version-container {
+    .current-version {
+      display:inline-block;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: space before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+  .channels {
+    display: flex;
+    justify-content: space-between;
+
+    .channel {
+      width: 300px;
+      display: inline-block;
+      *display: inline;
+      border: 12px solid @akeneoGreySuperLight;
+      transition:opacity 0.2s linear;
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: space before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 16]
```diff
@@ -0,0 +1,200 @@
...
+            border-bottom: 2px solid @akeneoGreySuperLight;
+          }
+        }
+        .locale {
+          padding: 0 10px;
+          .text-overflow();
+        }
+        .progress-cell {
+          width: 50%;
+          span.progress-info {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 17]
```diff
@@ -0,0 +1,200 @@
...
+      }
+
+      .locale-result {
+        font-size: @font-size-10;
+        width: 100%;
+        .clearfix();
+
+        tr {
+          line-height: 46px;
+          td {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 18]
```diff
@@ -0,0 +1,200 @@
...
+            border-bottom: 2px solid @akeneoGreySuperLight;
+          }
+        }
+        .locale {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 19]
```diff
@@ -0,0 +1,200 @@
...
+          padding: 0 10px;
+          .text-overflow();
+        }
+        .progress-cell {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line before

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 20]
```diff
@@ -0,0 +1,200 @@
...
+          width: 50%;
+          span.progress-info {
+            padding: 5px;
+            cursor: default;
+          }
+          .progress {
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: same for the rest of the file

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js [hunk 21]
```diff
@@ -10,62 +10,40 @@ define(
...
+            template: _.template(template),
+
+            _afterLoad: function() {
+                AbstractWidget.prototype._afterLoad.apply(this, arguments)
+                this.loadMore();
+            },
 
             events: {
-                'click a[data-toggle-channel]': 'toggleChannel'
+                'click #completeness-load-more': 'loadMore'
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: please don't use ids. There is no need to use ids

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js [hunk 22]
```diff
@@ -10,62 +10,40 @@ define(
...
+            loadMore: function (e) {
+                if (undefined !== e) {
+                    e.preventDefault();
+                }
 
-                var channel = $(e.currentTarget).data('toggle-channel');
-                this.$('tr[data-channel="' + channel + '"]').toggle();
-                this.$('a[data-toggle-channel="' + channel + '"] i')
-                    .toggleClass('icon-caret-right icon-caret-down');
+                var $nextChannels = $('#' + this.id).find('.channels:not(:visible)');
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: don't use ids

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js [hunk 23]
```diff
@@ -10,62 +10,40 @@ define(
...
+                if ($nextChannels.length) {
+                    $nextChannels.first().show();
+                }
+
+                if ($nextChannels.length <= 1) {
+                    $('#completeness-load-more').hide();
```
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: same

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 24]
```diff
@@ -0,0 +1,200 @@
...
+    }
+
+    .label {
+      min-height: 22px;
+      line-height: 22px;
+      padding: 0 4px;
+    }
+  }
+
+  background-color: white;
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: put it at the top of the block please

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 25]
```diff
@@ -0,0 +1,200 @@
...
+  &.quick-launchpad > .container {
+    padding-top: 30px;
+  }
+  .quick-launchpad-widget {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 9 above)
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: `> .clearfix` should be enough (no div, no &)

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 26]
```diff
@@ -0,0 +1,200 @@
...
+    margin-bottom: 20px;
+    &>div.clearfix {
+      overflow: hidden;
+      background-color: white;
+    }
+  }
+  h3 {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line pls

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 27]
```diff
@@ -0,0 +1,200 @@
...
+    margin: 0;
+    font-size: @font-size-18;
+    color: @color-new-purple-medium;
+    font-weight: normal;
+  }
+  &.header {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: idem

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 28]
```diff
@@ -0,0 +1,200 @@
...
+    background-color: @whiteDark;
+    padding: 15px;
+  }
+  .version-container {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: idem

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 29]
```diff
@@ -0,0 +1,200 @@
...
+      color: @akeneoSlateGrey;
+      cursor: pointer;
+    }
+  }
+}
+
+.completeness-widget {
+  color: @akeneoSlateGrey;
+
+  & > div {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: & is useless

Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: no class for this div?

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 30]
```diff
@@ -0,0 +1,200 @@
...
+    padding-top:15px;
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: missing space

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 16 above)
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: is the `span` discriminating here?

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 31]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+table.last-operations, table.proposal-widget {
+  width: 100%;
+  tr {
+    td, th {
+      padding: 0 10px;
+      line-height: 48px;
+      border-bottom: 2px solid @akeneoGreySuperLight;
+      &.text-right {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: Isn't it a bootstrap class?

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 32]
```diff
@@ -0,0 +1,200 @@
...
+    &:hover, &:focus {
+      background-color: @akeneoSlateGreyDark;
+      color: @akeneoLight;
+    }
+  }
+}
+
+table.last-operations, table.proposal-widget {
+  width: 100%;
+  tr {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: blank line pls

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 33]
```diff
@@ -0,0 +1,200 @@
...
+    td, th {
+      padding: 0 10px;
+      line-height: 48px;
+      border-bottom: 2px solid @akeneoGreySuperLight;
+      &.text-right {
+        text-align: right;
+      }
+    }
+    th {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: idem

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less [hunk 34]
```diff
@@ -0,0 +1,200 @@
...
+      background: @akeneoGreySuperLight;
+      font-size: @font-size-13;
+      border-bottom: 2px solid @akeneoSlateGrey;
+      text-align: left;
+    }
+  }
+  .label, .btn {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: idem

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html (hunk 1 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html]: Seen IRL

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html (hunk 2 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html (hunk 3 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html (hunk 4 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 6 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 5 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 7 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 8 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 9 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 10 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 11 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 12 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 13 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 14 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 15 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 16 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: Done.
Yes it is

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 17 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 18 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 19 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 20 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 24 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 25 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 26 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 27 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 28 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 29 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: It's more readable.
No class, I changed to *.

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 30 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 31 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: Overriden by us everywhere. Change it in pimui.less change everywhere, I prefer override it again.

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 32 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 33 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 34 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js (hunk 21 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js (hunk 22 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js (hunk 23 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: done

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 31 above)
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: ok

File: src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less (hunk 16 above)
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/css/dashboard.less]: I don't see where this class is used.

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html [hunk 35]
```diff
@@ -0,0 +1,32 @@
...
+<% _.each(chunkedChannels, function (channels) { %>
+    <div class="channels" style="display:none">
+    <% _.each(channels, function (channelResult, channel) { %>
+        <div class="channel">
+            <div class="header">
+                <%= channelResult.name %>
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html]: always escape with `<%-` ; )

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html [hunk 36]
```diff
@@ -0,0 +1,32 @@
...
+            <% }); %>
+            </table>
+        </div>
+    <% }); %>
+    </div>
+<% }); %>
+
+<% if (chunkedChannels.length > 1) { %>
+    <div class="text-center">
+        <div id="completeness-load-more" class="btn load-more"><%- _.__('pim_dashboard.widget.completeness.load_more') %></div>
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/completeness-widget.html]: id not used anymore

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html [hunk 37]
```diff
@@ -0,0 +1,44 @@
+<% if (!_.isEmpty(data)) { %>
+    <thead>
+        <tr>
+            <th>
+                <%= _.__('pim_dashboard.widget.last_operations.date') %>
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: We try to use the `__` translator everywhere, please pass it to the template and use it.

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html [hunk 38]
```diff
@@ -0,0 +1,44 @@
...
+                <a class="btn btn-large btn-download" href="javascript:void(0);"
+                data-id="<%= operation.id %>"
+                data-operation-type="<%= operation.type %>">
+                    <%= _.__('pim_dashboard.widget.last_operations.details') %>
+                </a>
+            </td>
+        </tr>
+    <% }); %>
+    </tbody>
+<% } else if (options.contentLoaded) {%>
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: Looks like it's always false, no?

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js [hunk 39]
```diff
@@ -10,62 +10,40 @@ define(
...
-                                    '<small><%= localeResult.complete %>/<%= channelResult.total %></small>',
-                                '</td>',
-                            '</tr>',
-                        '<% }); %>',
-                    '<% }); %>'
-                ].join('')
-            ),
+            template: _.template(template),
+
+            _afterLoad: function() {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/completeness-widget.js]: Travis fails here: missing space

**Comment by phaseinducer** [comment]
GTM after fix

File: src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html (hunk 38 above)
Comment (pierallard) [inline src/Pim/Bundle/DashboardBundle/Resources/public/templates/last-operations-widget.html]: nope, src/PimEnterprise/Bundle/DashboardBundle/Resources/public/js/proposal-widget.js:52

**Pull request merged**